hugo.yaml

```diff
@@ -1,6 +1,6 @@
 title: Docker Docs
 baseURL: https://docs.docker.com
-refLinksErrorLevel: ERROR
+refLinksErrorLevel: WARNING
```

Comment:
This is a temporary change to make the site build until we can vendor the latest version of docker/cli which contains a fix